### ISSUE TITLE
aria-hidden="true" should not have display: none !important

### ISSUE THREAD

**Issue by ducklet** [18F/web-design-standards#1120]
Hi,

I'm trying to use Font Awesome icons that are purely decorative and sit next to the text a screen reader needs.  Without `aria-hidden="true"`, a screen reader user will hear the same information twice, but `aria-hidden="true"` has been given the css rule `display: none !important`, which totally defeats its purpose.  Can you please fix this?  (Sorry it's a .pdf, but WebAIM recommends `aria-hidden="true"` for these icons: http://webaim.org/presentations/2014/myeducator/POUR.pdf)

Thanks!

**Comment by ducklet** [comment]
cc @edwelker

**Comment by maya** [comment]
Hi @ducklet, thanks for opening this issue! Can you point us to where you're seeing this rule in the code base?

**Comment by jbrucegsa** [comment]
Hi @ducklet, I'm curious about this as I really try to make thing accessible as much as possible. 

Can you show the markup that is causing it to be read by the screen reader?

For example:

<pre><code>
<i class="fa [icon]"></i>
</code></pre> 

or something else.

**Comment by edwelker** [comment]
https://github.com/18F/web-design-standards/blob/18f-pages-staging/src/stylesheets/core/_base.scss#L23

**Comment by scottaohara** [comment]
+1 for this issue.

aria-hidden is quite useful to hide content from assistive technology, but not visually (as ducket points out). 

maybe this should be replaced with the native [hidden] attribute, as that has some older cross browser issues where a display: none!important would be useful.

**Comment by ducklet** [comment]
Hi @jbrucegsa,

The issue is actually not that it will be read by a screen reader, but that I can't use it as intended because the `display: none !important` hides any element it's applied to from sighted users as well.  

Thanks for looking into it.

**Comment by nickbristow** [comment]
This is a good point @ducklet 

We should change how aria-hidden="true" is handled with CSS @maya 

We may need to add to update some things that were being hidden that way.

**Comment by maya** [comment]
Thanks for all the feedback! 

I agree with removing that from the CSS, and updating the instances that were relying on that to hide things in favor of something else (i.e. `hidden` attribute or `display: none!` on a per component basis like @scottaohara mentions in #1121).

**Comment by maya** [comment]
Good news, I searched the Draft Standards and the only place using `aria-hidden="true"` in this way is the accordion component. So if we can solve it there, we can remove this CSS.

**Comment by jhfrench** [comment]
FWIW: [The state of hidden content support in 2016](https://www.paciellogroup.com/blog/2016/01/the-state-of-hidden-content-support-in-2016/)

**Comment by maya** [comment]
@jhfrench Just wanted to note the `hidden` attribute is supported by IE 11 and above and this project supports IE 9 and above, so doesn't look like we can use it. http://caniuse.com/#search=hidden

**Comment by joshbruce** [comment]
@maya - Not sure implications, and not in a position to verify. The `hidden` attribute, if memory serves, is different than `aria-hidden`; so, #1192 still stands, yeah? Or, do we need to revisit?

**Comment by maya** [comment]
@joshbruce nope, we're all good, I just wanted to reply in response to https://github.com/18F/web-design-standards/issues/1120#issuecomment-210470332.
